modules/bmo-about-the-hostupdatepolicy-resource.adoc

```diff
@@ -0,0 +1,34 @@
+// This is included in the following assemblies:
+//
+// * installing/installing_bare_metal/ipi/ipi-install-post-installation-configuration.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="bmo-about-the-hostupdatepolicy-resource_{context}"]
+= About the HostUpdatePolicy resource
+
+You can use the `HostUpdatePolicy` resource to enable or disable applying live updates to the firmware settings, BMC settings, or BIOS settings of each bare-metal host. By default, the operator disables live updates to already provisioned bare-metal hosts by default.
+
+.The HostUpdatePolicy spec
+
+The `spec` section of the `HostUpdatePolicy` resource provides two settings:
+
+* `firmwareSettings`: This setting corresponds to the `HostFirmwareSettings` resource. 
+
+* `firmwareUpdates`: This setting corresponds to the `HostFirmwareComponents` resource.
+
+When set to `onPreparing`, you can only update the host during provisioning, which is the default setting. When set to `onReboot`, you can update a provisioned host by applying the resource and rebooting the bare-metal host. Then, follow the procedure for editing the `HostFirmwareSettings` or `HostFirmwareComponents` resource.
```

Comment:
For `onPreparing` and `onReboot`, API object references are supposed to be followed by a noun, such as spec, value, or resource. See the repo contribution guide: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references (specifically, the note admonition).

This feedback applies throughout the PR, I won't mention it in other instances.